frontend/src/metabase/visualizations/echarts/cartesian/model/dataset.ts

```diff
@@ -409,26 +409,20 @@ function getStackedValueTransfom(
   }));
 }
 
-function getStackedDataLabelTransform(
-  settings: ComputedVisualizationSettings,
-  seriesDataKeys: DataKey[],
-): ConditionalTransform {
-  return {
-    condition: settings["stackable.stack_type"] === "stacked",
```

Comment:
We don't really need to conditional check. It's fine if these values are included in questions where they are not needed, they'll just be ignored.